ISSUE_TEMPLATE.md

```diff
@@ -1,5 +1,4 @@
-- [ ] I tried using the latest `xxxx/xxxx.d.ts` file in this repo and had problems.
+- [ ] I tried using the `@types/xxxx` package and had problems.
 - [ ] I tried using the latest stable version of tsc. https://www.npmjs.com/package/typescript
 - [ ] I have a question that is inappropriate for [StackOverflow](https://stackoverflow.com/).  (Please ask any appropriate questions there).
-- [ ] I want to talk about `xxxx/xxxx.d.ts`.
-  - The authors of that type definition are cc/ @....
+- The authors (see `Definitions by:` in `index.d.ts`) are cc/ @....
```

Comment:
I'm unclear about what this should convey in the context of this list. Consider giving context for why a person may need this information.